tla/ccfraft.tla

```diff
@@ -546,7 +546,8 @@ AdvanceCommitIndex(i) ==
     /\ state[i] = Leader
     /\ LET
         \* We want to get the smallest such index forward that is a signature
-        new_index == SelectInSubSeq(log[i], commitIndex[i]+1, Len(log[i]), HasTypeSignature)
+        new_index == SelectInSubSeq(log[i], commitIndex[i]+1, Len(log[i]),
```

Comment:
Add a reference to the corresponding section in the Raft paper?

Reply:
Done